### ISSUE TITLE
TextField [IE]: when autoAdjustHeight is true, it shows flicker for scroll bar inside text field 

### ISSUE THREAD

**Issue by b-priyank** [OfficeDev/office-ui-fabric-react#6506]
### Bug Report 
In Internet Explorer, when autoAdjustHeight is true, it shows flicker for scroll bar inside text field. This happens when text exceeds available space and makes text field increase its height on enter.
 
- __Package version(s)__: 6.74.1
- __Browser and OS versions__: IE with windows 10
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: Azure DevOps
 
#### Describe the issue:
 
In Internet Explorer, when autoAdjustHeight is true, it shows flicker for scroll bar inside text field. This happens when text exceeds available space and makes text field increase its height on enter.
 
#### Actual behavior:
 
It shows flicker for scroll bar inside text field
 
#### Expected behavior:
 
It should not show flicker for scroll bar inside text field

**Comment by Vitalius1** [comment]
@b-priyank by flicker the scrollbar inside TextField you mean it appears for a split second when the text wraps on the next line but then disappears?

**Comment by b-priyank** [comment]
@Vitalius1 yes, it appears for split second and disappears once text field expands.

**Comment by mikewheaton** [comment]
I captured a video of the issue, and here's the scroll bar that appears briefly:
![image](https://user-images.githubusercontent.com/1382445/47102212-f5250380-d1f0-11e8-9b48-7f7a1973dc01.png)

It looks to be a timing issue, where our component doesn't call `_adjustInputHeight` until after the text has already been typed and the browser has added the scroll bar. Given that there should never be a scroll bar (the field continues growing indefinitely) the simplest fix seems to be adding `overflow: hidden` to the field. But I also see there's a [line](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/TextField/TextField.base.tsx#L517) that was added to avoid vertical scroll bars, and increasing that value might help.
